src/App.tsx

```diff
@@ -57,10 +57,15 @@ export const App = async (props: AppProps, container?: HTMLElement) => {
             referrer: window.location.href
           }),
         }
-      );
+      )
       const body = await response.json();
       console.log("Token response", body);
 
+
+      //FEEDYOU - Loader
+      document.querySelector("body").classList.add("loaded")
```

Comment:
Can we please use classes "feedbot-enabled" or "feedbot-disabled" (any of them on body should cause spinned to be hidden) which are already being added on body instead of the new "loadded" one?